tests/data/model-validation.ts

```diff
@@ -1559,15 +1569,16 @@ type document
 `,
     expectedError: [
       {
-        endColumn: 10,
+        endColumn: 48,
         endLineNumber: 8,
         extraInformation: {
-          error: "type-wildcard-relation",
+          error: "tupleuserset-not-direct",
+          relation: "viewer",
         },
-        message: "type restriction `document:*` cannot contain both wildcard and relation",
+        message: "`parent` relation used inside from allows only direct relation.",
         severity: 8,
         source: "ModelValidationError",
-        startColumn: 0,
+        startColumn: 42,
```

Comment:
The column numbers seem off - shouldn't this be 41? Looking at Monaco Diagnostics, they are 0 based too

Reply:
At the moment this package just copies the line and column data from language, so yeah the errors here are one based rather than zero based.

I could fix this package to correctly change to zero based, but that would then be reverted once openfga/language/pull/223 is released. WDYT to just leaving as is until the next language release which will correctly start reporting zero based and we can update the line numbers in tests as appropriate?